js/src/components/contact-information/index.js

```diff
@@ -127,7 +116,7 @@ const ContactInformation = ( { onPhoneNumberVerified } ) => {
 				<PhoneNumberCard
 					view="setup-mc"
 					phoneNumber={ phone }
-					initEditing={ initEditing }
+					initEditing={ null }
```

Comment:
💅  We can just omit this props. Since the default value is `null`

Reply:
Removed in https://github.com/woocommerce/google-listings-and-ads/commit/e3f4d7938cfce994e3a4c81d92ce5059de10a1ed.